packages/elastic_agent/manifest.yml

```
@@ -1,14 +1,14 @@
 name: elastic_agent
 title: Elastic Agent Metrics
-version: 1.2.2
+version: 1.2.3
```

I would bump to a new minor, in case it is needed to continue releasing fixes only for 7.x at some moment.
```suggestion
version: 1.3.0
```

---

Good idea. Done.

---

I am thinking now that my comment didn't make much sense :sweat_smile: If a new package is needed for 7.x at some moment it will need to have a greater version than this package, so `1.2` minor cannot be used for that.

---

Good point, should have catched that on my end 🤦 We will find a way if needed.